src/server/readpool/mod.rs

```diff
@@ -305,10 +268,15 @@ mod tests {
         // full
         assert!(spawn_long_time_future(&read_pool, 8, 100).is_err());
 
-        assert_eq!(rx.recv().unwrap(), Ok(2));
-        assert_eq!(rx.recv().unwrap(), Ok(3));
-        assert_eq!(rx.recv().unwrap(), Ok(7));
-        assert_eq!(rx.recv().unwrap(), Ok(4));
+        // the recv order maybe: "Ok(2)Ok(4)Ok(7)Ok(3)" or “Ok(2)Ok(3)Ok(4)Ok(7)” or “Ok(2)Ok(4)Ok(3)Ok(7)”
+        print!("{:?}", rx.recv().unwrap());
```

Comment:
I think we should use assert_eq, not print

Reply:
Before the recv order was certainly Ok(2)Ok(3)Ok(7)Ok(4), but now it's order changes every runs. So I am not sure whether it is a problem.

Reply:
Then let's not check the recv order any more. Let's only check whether or not full is returned, since futurepool already has complete tests. This is a work-stealing pool and the scheduling order is not as predictable as the previous one.